File: C Functions Task/todo3/app3.java
public class Main
{
	public static void main(String[] args) {
	   	System.out.print("Result:");
		//function caller
	   	Biggest(10,20);
	}
	//called function
	public static void Biggest(int a,int b){
	    int result=a>b?a:b;
	    System.out.println(result);
	}
}

File: C Functions Task/todo2/app3.c
#include <stdio.h>
//called function
void Biggest(int a,int b){
    int result=a>b?a:b;
    printf("%d",result);
}

int main()
{
    int a=10;
    int b=20;
    //function caller
    Biggest(a,b);
    return 0;
}
